CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
 All notable changes to this project will be documented in this file.
 
-# [v0.20.3](https://github.com/cryptape/cita-sdk-swift/compare/v0.20...v0.20.3) (2019-01-21)
+# [v0.21](https://github.com/cryptape/cita-sdk-java/compare/v0.20.3...v0.21) (2019-02-18)
```

Comment:
@duanyytop Why remove changeling for v0.20.3 🤔 ?

Reply:
You mean to list all versions change log? I get it .

Reply:
Right.